pages_/my-turns.tsx

```diff
@@ -13,12 +14,62 @@ import Router from 'next/router';
 import Layout from 'src/components/Layout';
 import graphqlClient from 'src/graphqlClient';
 import { getErrorCodeFromApollo } from 'src/utils';
+import Spinner from 'src/components/Spinner';
 
 // type Props = {};
 
 const MyTurns = () => {
   const { t } = useTranslation();
 
+  const [myTurns, setMyTurns] = useState<any>();
+  const [error, setError] = useState();
+
+  const getTurns = async () => {
+    const MY_TURNS = /* GraphQL */ `
+      query MyTurns {
+        myTurns {
+          shopId
+          shopName
+          turnInfo {
+            id
+            status
+            turn
+          }
+        }
+      }
+    `;
+
+    try {
+      const res = await graphqlClient.request(MY_TURNS);
+      setMyTurns(res.myTurns);
+    } catch (error) {
+      setError(error);
+    }
+  };
+
+  // fetch data
+  useEffect(() => {
+    getTurns();
+
+    const interval = setInterval(() => {
+      getTurns();
+    }, 10_000);
```

Comment:
Do you think an interval is necessary here? The only two actions that can change in "real-time" that list is the shop owner skipping or cancelling the turn, so we are adding another costly db round trip just for those 2 cases which are not essential.